react/features/filmstrip/components/web/Filmstrip.js

```diff
@@ -329,7 +322,7 @@ class Filmstrip extends PureComponent <Props, State> {
         }
         case LAYOUTS.TILE_VIEW: {
             if (_stageFilmstrip && _visible) {
-                filmstripStyle.maxWidth = `calc(100% - ${_verticalViewMaxWidth}px - ${_chatOpen ? CHAT_SIZE : 0}px)`;
```

Comment:
Why is this no longer needed here? IS it accounted for by its container perhaps?

Reply:
Yes. Because of this 3eafaeeedd1d29960913c55acaebab6e1d6eb05e commit. The chat is no longer positioned absolute so now it influences the container width

Reply:
Wonderful!